wails-frontend/src/pages/importFile.tsx

```diff
@@ -23,7 +24,7 @@ const ImportFile = () => {
 
   const baselineStr = () =>
     account
-      ? `Importing account ${account.nickname}`
+      ? `Selected ${account.nickname}'s account`
       : 'Select an account file to import';
   const applyStr = () => (account ? 'Import' : 'Select a file');
```

Comment:
Why these need to be a function!? 🤔 

Can't we just assign as variable and the in the code access the variable (baselineStr or applyStr)!?

BTW these both names are a mystery... can't we be more explicit what they are!? 🤔

Reply:
I have no say, I would recommend seeing that with @peterjah  who set it up !

Reply:
give it a try without function, const instead and let'see if it work too and if it's better

Reply:
also addressed the naming ! applyStr is now importStr

Reply:
ok for me